woocommerce/woocommerce-gutenberg-products-block
# Add types to hooks

This PR adds types to some hooks to learn more TS.

`assets/js/base/hooks/cart/use-store-cart-coupons.ts`
`assets/js/base/hooks/cart/use-store-cart-item-quantity.ts`
`assets/js/base/hooks/cart/use-store-cart.ts`
`assets/js/base/hooks/customer/use-customer-data.ts`
`assets/js/base/hooks/customer/utils.ts`
`assets/js/base/hooks/use-container-queries.ts`
`assets/js/base/hooks/use-previous.ts`
`assets/js/base/hooks/use-shallow-equal.ts`
`assets/js/base/hooks/use-store-add-to-cart.ts`
`assets/js/base/hooks/use-throw-error.ts`
`assets/js/base/hooks/use-trigger-fragment-refresh.ts`
`assets/js/type-defs/hooks.ts`

### Testing instructions
- Smoke test cart and checkout to make sure there is no regression because I renamed some variables.
